index.html

```diff
@@ -11,151 +11,236 @@
-        </div>
-      </div>
-      <div class="card">
-        <img src="image/Snapshoot-Portfolio-(2).png" alt="A photo of a website page">
-        <div class="card-content">
-          <h2 class="card-heading">Tonic</h2>
-          <ul class="tool1">
-            <li>CANOPY</li>
-            <li><img src="image/Counter.png" alt="dot"></li>
-            <li class="fint">Back End Dev</li>
-            <li><img src="image/Counter.png" alt="dot"></li>
-            <li class="fint">2015</li>
-          </ul>        
-          <div class="detail-style1">A daily selection of privately<br>reads:no account or<br>sign-up required</div>
-          <ul class="tool2">
-            <li><a href="#">html</a></li>
-            <li><a href="#">css</a></li>
-            <li><a href="#">javascript</a></li>
-          </ul>
-          <button type="button" class="button detail-style2">See Project</button>
         </div>
-      </div>
-      <div class="card">
-        <img src="image/Snapshoot-Portfolio-(3).png" alt="A photo of a website page">
-        <div class="card-content">
-          <h2 class="card-heading">Multi-Post<br>Stories</h2>
-          <ul class="tool1">
-            <li>CANOPY</li>
-            <li><img src="image/Counter.png" alt="dot"></li>
-            <li class="fint">Back End Dev</li>
-            <li><img src="image/Counter.png" alt="dot"></li>
-            <li class="fint">2015</li>
-          </ul>        
-          <div class="detail-style1">A daily selection of privately<br>reads:no account or<br>sign-up required</div>
-          <ul class="tool2">
-            <li><a href="#">html</a></li>
-            <li><a href="#">css</a></li>
-            <li><a href="#">javascript</a></li>
-          </ul>
-            <button type="button" class="button detail-style2">See Project</button>
+      </section>
+      <section class="card-section">
+        <div class="cards">
+          <div class="card first">
+            <img src="image/Snapshoot-Portfolio-4.png" alt="a baby" class="web">
+            <img src="image/Snapshoot-Portfolio.png" alt="A photo of a website page" class="mob">
+            <div class="card-content">
+              <h2 class="card-heading">Tonic</h2>
+              <ul class="tool1">
+                <li>CANOPY</li>
+                <li><img src="image/Counter.png" alt="dot"></li>
+                <li class="fint">Back End Dev</li>
+                <li><img src="image/Counter.png" alt="dot"></li>
+                <li class="fint">2015</li>
+              </ul>        
+              <p class="detail-style1">A daily selection of privately reads:no account or sign-up required</p>
+              <ul class="tool2">
+                <li><a href="#">html</a></li>
+                <li><a href="#">css</a></li>
+                <li><a href="#">javascript</a></li>
+              </ul>
+              <button type="button" class="button detail-style2">See Project</button>
+            </div>
```

Comment:
Kindly add the image for the first project since it does not appear in the desktop version.
![image](https://user-images.githubusercontent.com/24703171/155534823-3706b114-21e9-460a-b17f-511502c0c8b9.png)